### ISSUE TITLE
Serialization libraries?

### ISSUE THREAD

**Issue by dieggsy** [CodyReichert/awesome-cl#491]
There doesn't appear to be a section on serialization.

Maybe a whole extra section isn't necessary, but I think it would be nice to at least mention [cl-store](https://cl-store.common-lisp.dev/).

**Comment by vindarel** [comment]
correct, there's improvements to be done in that area. There are some solutions under "persistent object DB" (even if those are not persistent DBs): cl-naive-store and clache (that uses cl-store). I think at the time I didn't mention cl-store because I didn't like its documentation, and IIRC clache is a bit more high-level.

**Comment by dieggsy** [comment]
For what it's worth, cl-store does appear to have had [a bit more recent development](https://github.com/skypher/cl-store/commits/master/) than clache and I find its API more straightforward and focused. If someone just wants to serialize to disk, I think cl-store makes sense (and pulls in fewer dependencies)

**Comment by hxzrx** [comment]
For binary serialization, I recommend [cl-conspack](https://github.com/conspack/cl-conspack), which was inspired by MessagePack. The usage is very easy and it's very convenient to transfer data over networks, and some impls in other languages are also available.
